Basic Rules.txt

```diff
@@ -359,17 +359,42 @@ determined by your character class and an associated primary stat. Your class ma
 Some class feats and other abilities can increase the number of Nanites you can hold and use at a given time. 
 Also, Unlike Health, Nanites regenerate during combat at a rate of 3 Nanites per turn. 
 	
+
 ===== Assigning Stats to a New Character =====
 	When making a new character, you'll have to assign their Base stats.
 This will tell you a lot about your character. When assigning your stats,
 no stat can be higher than a 9 before applying racial/species bonuses and
 no stat can be lower than a 1 after applying racial/species bonuses. 
-Additionally, even after race/species bonuses, feats, equipment and base stat 
+Additionally, even after race/species bonuses, feats, equipment, and base stat 
 bonuses from leveling, no stat can ever be over 15 until your character is 
 level 15. After level 15, the limit for stat points is 20. The number of 
 initial base stat points that you have to apply is listed with your Race or 
 Species. See chapter 1: Races if you haven't yet. 
 
+===== Assigning A Class to your Character =====
+	Most Characters start at level 1 so the Player will choose a Class for that 
+Character at that time. A Character will only ever have one Class at a time, and
+under normal circumstances, cannot change their class once it is picked. 
+	On occasion (especially for new Players), a character may start at level 0 
+with no Class. This is usually for a tutorial mission and a Class is selected
+when the Character has completed the tutorial and leveled up to level 1. 
+
+===== Weapon Proficiencies =====
+	Each Class comes with a specific list of Weapon Proficiencies. If your 
+Character has a proficiency with a type of weapon, then they can use that weapon

```

Comment:
If Proficiency is a glossaried term, then I believe our style mandates we capitalize all uses of it (as a game term)?